src/aws-elasticache/user.ts

```
@@ -344,14 +347,19 @@ export class PasswordUser extends BaseUser implements IPasswordUser {
     return new Import(scope, id);
   }
 
+  /**
+   * The secret containing the generated password
+   */
+  public get secret(): aws_secretsmanager.ISecret | undefined {
```

If the construct doesn't generate a secret, Could we raise an error?

---

Also property name was renamed to `generatedSecret`
https://github.com/open-constructs/aws-cdk-library/pull/65/commits/90d65a1405a25a4726a9a81cfd028cc3847ccf62